backend/resolver/resolver.go

```diff
@@ -16,7 +16,11 @@ import (
 	resolverv1 "github.com/lyft/clutch/backend/api/resolver/v1"
 )
 
-const OptionAll = "__ALL__"
+const (
+	OptionAll = "__ALL__"
+	// The maximum amount of results to return for the auto complete api
+	AutoCompleteAPILimit = 20
```

Comment:
i would go ahead and make this an optional parameter that threads through and then default it to this value at the module level rather than using a const